Makefile

```diff
@@ -1,4 +1,5 @@
 DOCKER_REPO             ?= prombench
+DOCKER_IMAGE_NAME		?= prombench
```

Comment:
We'll need to build other images eg, fake-webserver, scaler etc. so I think this will be best passed as a variable to circleci.

eg. https://github.com/prometheus/prometheus/blob/2ccc48adc6eaf2623311366ab7748475c277f443/.circleci/config.yml#L92-L99

Reply:
correct